### PR TITLE
AWS Lambda context type hinting

### PR DESCRIPTION
**Pull request by emcpow2**
- Add type hints for lambda handler context parameter

### CODE REVIEW

**Comment by emcpow2** [comment]
how about this PR?

File: requirements.txt [hunk 1]
```diff
@@ -9,6 +9,8 @@ moto
 mypy
 brotli
 brotli-asgi
+awslambdaric
+awslambdaric-stubs
```
Comment (koxudaxi) [inline requirements.txt:13]: @emcpow2 
Thank you for creating this PR.
Did we need `awslambdaric` for type-checking?
I guess `awslambdaric-stubs` provides type stub.

```python
from mangum import Mangum

Mangum()({}, {})
```

```bash
$ mypy main.py                                                                                                                                     ✘ 1 
main.py:3: error: Missing positional argument "app" in call to "Mangum"
main.py:3: error: Argument 2 to "__call__" of "Mangum" has incompatible type "Dict[<nothing>, <nothing>]"; expected "LambdaContext"
Found 2 errors in 1 file (checked 1 source file)

$ pip list |grep aws                                                                                                                             
awslambdaric-stubs       0.1.0

```

Comment (emcpow2) [reply requirements.txt:13]: valid concern

removing redundant dependency

**Comment by koxudaxi** [comment]
Thank you very much!!

**Pull request merged**